### PR TITLE
Issues/fms pro page niggles

### PR DESCRIPTION
**Pull request by wrightmartin**
Latest updates to the fixmystreet professional pages

fixmystreet professional fixmystreet

### CODE REVIEW

File: web/cobrands/fixmystreet.com/fmsforcouncils.scss [hunk 1]
```diff
@@ -204,6 +225,10 @@ $fms-green: #62b356;
     margin-top: 18px;
   }
 
+  .councils-features__screenshot {
```
Comment (zarino) [inline web/cobrands/fixmystreet.com/fmsforcouncils.scss]: Why’s this empty ruleset here?

File: web/cobrands/fixmystreet.com/fmsforcouncils.scss [hunk 2]
```diff
@@ -492,6 +528,19 @@ $fms-green: #62b356;
     }
   }
 
+  .pricing-table__prices {
+    td {
+      background-color: #E65376;
```
Comment (zarino) [inline web/cobrands/fixmystreet.com/fmsforcouncils.scss]: We use `#E65376` a lot in this file. I wonder whether you could make it into a variable up top, like the others? `$fms-pink` perhaps?

File: web/cobrands/fixmystreet.com/fmsforcouncils.scss [hunk 3]
```diff
@@ -183,16 +186,34 @@ $fms-green: #62b356;
     }
     h3 {
       font-weight: normal;
-      margin-left: 75px;
       margin-bottom: 0.25em;
     }
     p {
       color: #777;
```
Comment (zarino) [inline web/cobrands/fixmystreet.com/fmsforcouncils.scss]: Maybe it’s just my screen but I find this `#777` text kinda hard to read on the `#F7F6F5` background from `.councils-sales--benefits`. It’s a 4.1 contrast ratio, which is _almost_ the 4.5 needed for WCAG AA, but I think maybe the text being super small isn't helping either.

**Comment by zarino** [comment]
Oh, also, would be nice to change the commit message for ad2355b from ```[fixmystreet.com]Change to pricing table``` to ```[fixmystreet.com] Change to pricing table``` :-)

**Comment by zarino** [comment]
> Oh, also, would be nice to change the commit message for ad2355b…

Or, to be fair, you could probably squash all of this into one commit.

File: web/cobrands/fixmystreet.com/fmsforcouncils.scss (hunk 1 above)
Comment (wrightmartin) [inline web/cobrands/fixmystreet.com/fmsforcouncils.scss]: That was there as I planned to have code ready for screenshots to be added, didn't get round to it in the end though, will remove

**Pull request merged**